packages/astro/src/core/dev/index.ts

```diff
@@ -58,26 +58,13 @@ export default async function dev(config: AstroConfig, options: DevOptions = { l
 /** Dev server */
 export class AstroDevServer {
   app = connect();
-  httpServer: http.Server | undefined;
-  hostname: string;
-  port: number;
-  private config: AstroConfig;
-  private logging: LogOptions;
-  private manifest: ManifestData;
-  private mostRecentRoute?: RouteData;
-  private site: URL | undefined;
-  private devRoot: string;
-  private url: URL;
-  private origin: string;
-  private routeCache: RouteCache = {};
-  private viteServer: vite.ViteDevServer | undefined;
```

Comment:
**Non-blocking nit** Why move these? I tend to prefer all the class properties defined up top, especially ones that are intitialized like `routeCache = {}`.

Reply:
🙏 Thank you for catching this! I meant to put them back. The only change should have been removing `private` since they were not actually private.

Reply:
This has been fixed.